client/src/main/java/com/paypal/selion/platform/utilities/WebDriverWaitUtils.java

```diff
@@ -67,7 +64,6 @@ private static void waitForCondition(ExpectedCondition<?> condition, long timeou
      */
     public static void waitUntilElementIsClickable(final String elementLocator) {
         logger.entering(elementLocator);
-        Preconditions.checkState(Grid.driver() != null, INVALID_STATE_ERR_MSG);
```

Comment:
Why not still do a Grid.driver() here just to verify that the driver is initialized to the state we expect here.  Is there another way to confirm the driver is initialized here and in other places?